test-memgraph.yaml

```diff
@@ -0,0 +1,168 @@
+---
```

Comment:
Is there a reason why you have included the `dry-run` in the PR? This looks like it should be deleted.

Reply:
No reason it's just a separator not a dry-run imo.